cluster/eval_server/README.md

```diff
@@ -0,0 +1,30 @@
+
+# Minigo Eval Server
+
```

Comment:
one setence summary of problem this is solving; a README for something called 'eval' should at a minimum describe what eval stands for and what it evaluates.  See also https://lamport.azurewebsites.net/pubs/state-the-problem.pdf

Reply:
Good idea. I added some stuff.